crypto/x86_64cpuid.pl

```diff
@@ -145,8 +145,19 @@
 	or	\$0x40000000,%edx	# set reserved bit#30 on Intel CPUs
 	and	\$15,%ah
 	cmp	\$15,%ah		# examine Family ID
-	jne	.Lnotintel
+	jne	.LnotP4
 	or	\$0x00100000,%edx	# set reserved bit#20 to engage RC4_CHAR
+.LnotP4:
+	cmp	\$6,%ah
+	jne	.Lnotintel
+	and	\$0x0ffff0f0,%eax
```

Comment:
This line seems off. Both comparisons will always be not equal because `6` lines up with a `0`, not an `f`. Did you mean `0x0fff0ff0`?

(Checking `6` is also redundant with lines 151 and 152.)